tests/generative/variable_strategies.py

```diff
@@ -112,6 +113,11 @@ def add(draw, type_, frame):
     def count(_type, _frame):
         return st.builds(AggregateByPatient.Count, any_frame())
 
+    @st.composite
+    def is_null(draw, _type, frame):
+        type_ = draw(any_type())
+        return Function.IsNull(draw(series(type_, frame)))
```

Comment:
I think this is what IsNull needs - a series of any type, but it can be a one or many rows per patient series, so we can just pass on `frame` without reducing it with `one_row_per_patient_frame_or`?

Reply:
Yes, I think this is fine.